NMDSdevopsServiceAdm/DataEngineering
# Fix pipfile dependency mismatch

# Description
match black version to one within circleci config, and fixed versioning of numpy
No need for a change to documentation as this change doesn't change any instructions in the documentation

# How to test
To see the breakage for numpy, try to run `pipenv install --dev` from a fresh install of the repository. For the numpy version in this change, the old one (1.19.5), you will get an error that locks the subprocess, mentioning and error with building the wheel for numpy.

Change the version of numpy to 1.26.5 and re-run, and this step for setup then works.
You can also see it correctly install under pandas too by running the following after completion, to see the version that's installed.
`pipenv graph --reverse`

# Developer checklist
